.prettierrc

```diff
@@ -4,5 +4,6 @@
   "trailingComma": "es5",
   "bracketSpacing": true,
   "jsxBracketSameLine": false,
-  "useTabs": false
+  "useTabs": false,
+  "printWidth": 80
```

Comment:
we talked in chat, better remove for now, and then run eslint script, I can do it. better after merge to master, to prevent confilicts

Reply:
ok will remove, that because my VSCode shoutin about thant , maybe is good to apply all the rules at once somehow)